protos/protos/public/registry/RegistryService.proto

```diff
@@ -0,0 +1,379 @@
+syntax = "proto3";
+
+package ai.verta.registry;
+option go_package = "github.com/VertaAI/protos-all/protos/gen/go/protos/private/registry";
+
+option java_multiple_files = true;
+
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+import "common/CommonService.proto";
+import "modeldb/versioning/Environment.proto";
+
+message RegisteredModel {
+    // internal id
+    uint64 id = 1;
+
+    // Unique name for the model within the workspace.
+    string name = 2;
+
+    // Timestamp recorded when this ``registered_model`` was created.
+    int64 time_created = 3;
+
+    // Timestamp recorded when metadata for this ``registered_model`` was last updated.
+    int64 time_updated = 4;
+
+    // Description of this ``registered_model``.
+    string description = 5;
+
+    // Labels added to the registered models
+    repeated string labels = 6;
+
+    // Visibility of Registered Models
+    common.VisibilityEnum.Visibility visibility = 7;
```

Comment:
There is no VisibilityEnum in CommonService

Reply:
fixed